en/guide/installation.md

```diff
@@ -43,11 +45,12 @@ $ mkdir <project-name>
 $ cd <project-name>
 ```
 
-*Info: replace project-name by the name of the project.*
+*Info: replace `<project-name>` by the name of the project.*
```

Comment:
The correct part to remlace is `<project-name>` with `<` `>` remplaced. Because it's a code part, we use tick.